tools/mo/openvino/tools/mo/back/preprocessing.py

```diff
@@ -80,13 +82,26 @@ def check_keys_valid(ov_function: Model, keys: list, search_outputs: bool):
                 nodes_used[ov_node] = name
                 node_found = True
                 break
+            elif name == ov_node.get_node().get_friendly_name():
+                assert len(ov_node.get_tensor().get_names()) > 0, 'Node must have at least one tensor name'
+                new_name = list(ov_node.get_tensor().get_names())[0]
+                rename_dict[name] = new_name
+                nodes_used[ov_node] = name
+                node_found = True
+                break
+
 
         if not node_found:
             if not search_outputs:
                 raise Error('Input with name {} wasn\'t found! {}'.format(name, refer_to_faq_msg(83)))
             else:
                 raise Error('Input/Output with name {} wasn\'t found! {}'.format(name, refer_to_faq_msg(83)))
 
+    for name, new_name in rename_dict.items():
+        if name in dict_to_validate:
```

Comment:
i guess `name` is always in 'dict_to_validate` here, so we can add 'assert' instead